src/django_components/templatetags/component_tags.py

```diff
@@ -728,32 +375,10 @@ def parse_component_with_args(
     return component_name, context_args, context_kwargs
 
 
-def safe_resolve(context_item: FilterExpression, context: Context) -> Any:
-    """Resolve FilterExpressions and Variables in context if possible.  Return other items unchanged."""
-
-    return context_item.resolve(context) if hasattr(context_item, "resolve") else context_item
-
-
 def is_wrapped_in_quotes(s: str) -> bool:
     return s.startswith(('"', "'")) and s[0] == s[-1]
 
 
-def is_dependency_middleware_active() -> bool:
-    return getattr(settings, "COMPONENTS", {}).get("RENDER_DEPENDENCIES", False)
-
-
-def norm_and_validate_name(name: str, tag: str, context: Optional[str] = None) -> str:
```

Comment:
`norm_and_validate_name` isn't being used anywhere and neither is mentioned in any discussion/issue, I so removed it.